api/v1/GameSession.php

```diff
@@ -74,7 +74,7 @@ public static function getRequestApiRootAsync(): PromiseInterface
             $deferred = new Deferred();
             return resolveOnFutureTick($deferred, $GLOBALS['RequestApiRoot'])->promise();
         }
-        $apiRoot = preg_replace('/(.*)\/api\/(.*)/', '$1/', $_SERVER["REQUEST_URI"]);
+        $apiRoot = preg_replace('/(.*)\/(api|_profiler)\/(.*)/', '$1/', $_SERVER["REQUEST_URI"]);
```

Comment:
such that api and _profiler use the same mechanism. In this way the api root usage will function for both